fontforge/sfd.c

```diff
@@ -5619,6 +5747,22 @@ return( NULL );
 	    else
 		lasti->next = img;
 	    lasti = img;
+	} else if ( strmatch(tok,"Image2:")==0 ) {
+	    enum MIME mime = SFDGetImage2MIME(sfd);
+	    if (mime == UNKNOWN) exit(1);
+#ifndef _NO_LIBPNG
+	    int ly = current_layer;
+	    if ( !multilayer && !sc->layers[ly].background ) ly = ly_back;
+	    img = SFDGetImagePNG(sfd);
+	    if ( sc->layers[ly].images==NULL )
+		sc->layers[ly].images = img;
+	    else
+		lasti->next = img;
+	    lasti = img;
+#else
+        IError("PNG in SFD although we don't have libpng, cannot read SFD.");
+        exit(1);
```

Comment:
Sorry for ignorance, but how? Just read and throw away all the numbers then all the enc85 lines? Should I make a new fn, `FastForwardPastEnc85`?

Reply:
Oh, I missed it from my commit, but you can use `SFDConsumeUntil` to read until the `EndImage2` tag.

Reply:
It might be safer to do SFDConsumeUntil because it's technically possible for the text to match a tag, I think...just extremely unlikely. So I'll do that, one moment please :-)

Reply:
Thinking about it more, it's impossible as IHDR must always be the end, and no line but the end has any chance of matching, so no need

Reply:
I wouldn't mind using `SFDConsumeUntil` for it to be explicit, but up to you